crates/primitives/src/datalake/transactions/mod.rs

```diff
@@ -24,16 +24,15 @@ mod tests {
     #[test]
     fn test_transactions_datalake() {
         let encoded_datalake= "0x000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000f42400000000000000000000000000000000000000000000000000000000000000001000000000000000000000000000000000000000000000000000000000000000a0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000101010100000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000000020100000000000000000000000000000000000000000000000000000000000000";
-
+        let sampled_property = TransactionsCollection::from_str("tx.nonce").unwrap();
```

Comment:
we can think of creating a builder patter for this later
TransactionsCollection::from_str("tx.nonce")
so we dont specify some not elegant string like tx.nonce nothing blocks me to make a mistake here like tx.nomce

Reply:
it is enum hehe i updated it